src/java.desktop/share/classes/sun/print/PathGraphics.java

```diff
@@ -1132,6 +1133,9 @@ protected BufferedImage getBufferedImage(Image img) {
             // VI needs to make a new BI: this is unavoidable but
             // I don't expect VI's to be "huge" in any case.
             return ((VolatileImage)img).getSnapshot();
+        } else if (img instanceof MultiResolutionImage) {
+            return convertToBufferedImage((MultiResolutionImage) img,
```

Comment:
guess this casting is not needed..

Reply:
It is required, without casting it throws `incompatible type error`.